### PR TITLE
Make live logs reading work for "other" k8s executors

### PR DESCRIPTION
**Pull request by dstandish**

### CODE REVIEW

File: airflow/utils/log/file_task_handler.py [hunk 1]
```diff
@@ -132,6 +132,19 @@ def _render_filename(self, ti: TaskInstance, try_number: int) -> str:
     def _read_grouped_logs(self):
         return False
 
+    @staticmethod
+    def _try_read_from_k8s(queue):
```
Comment (uranusjr) [inline airflow/utils/log/file_task_handler.py]: This does not actually read from k8s, should it be renames to, say, `should_read_from_k8s`?

Comment (dstandish) [reply airflow/utils/log/file_task_handler.py]: yeah it's like "_should maybe_ try read from k8s"

Comment (dstandish) [reply airflow/utils/log/file_task_handler.py]: ok @uranusjr ready for another look... and added test

File: airflow/utils/log/file_task_handler.py [hunk 2]
```diff
@@ -173,7 +190,8 @@ def _read(self, ti: TaskInstance, try_number: int, metadata: dict[str, Any] | No
                 log = f"*** Failed to load local log file: {location}\n"
                 log += f"*** {str(e)}\n"
                 return log, {"end_of_log": True}
-        elif conf.get("core", "executor") == "KubernetesExecutor":
+
```
Comment (dstandish) [inline airflow/utils/log/file_task_handler.py]:
```suggestion
```

**Review by jedcunningham — APPROVED** [review]

**Pull request merged**